samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -78,6 +78,9 @@ See :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide for details.
    :start-after: nrf_desktop_fastpair_important_start
    :end-before: nrf_desktop_fastpair_important_end
 
+.. important::
+   To support the input device use case, you must select the ``input device`` option in the **Device Type** field during device model registration.
```

Comment:
```suggestion
.. note::
   To support the input device use case, you must select the **input device** option in the **Device Type** field when registering your device model.
```
I'd suggest to change this into a regular note

Reply:
ok